python/cudf/cudf/core/index.py

```diff
@@ -325,7 +325,7 @@ def to_pandas(self):
         >>> type(idx)
         <class 'cudf.core.index.GenericIndex'>
         """
-        return pd.Index(self._values.to_pandas(), name=self.name)
+        return pd.Index(self._values.to_arrow().to_pandas(), name=self.name)
```

Comment:
Can add `copy=False` to this constructor. Also may want to just use `self.values_host` instead of `to_arrow().to_pandas()`